mltsp/docker_scripts/docker_extract_custom_feats.py

```diff
@@ -22,14 +25,16 @@ def extract_custom_feats():
     """
     # load pickled ts_data and known features
     with open(
-        "/home/mltsp/copied_data_files/features_already_known_list.pkl",
-        "rb") as f:
+        "/home/copied_data_files/features_already_known_list.pkl", "rb") as f:
         features_already_known_list = pickle.load(f)
 
-    # script has been copied to the following location:
-    script_fpath = ("/home/mltsp/mltsp/custom_feature_scripts/"
+    #shutil.copy(
+    #    "/home/copied_data_files/custom_feature_defs.py",
+    #    "/home/mltsp/mltsp/custom_feature_scripts/custom_feature_defs.py")
+    #script_fpath = ("/home/mltsp/mltsp/custom_feature_scripts/"
+    #                "custom_feature_defs.py")
+    script_fpath = ("/home/copied_data_files/"
```

Comment:
It's just an unusual place to put data files (in terms of unix).
Perhaps even creating "/data" would be better?